docs/architecture/app-bootstrap.md

```
@@ -0,0 +1,207 @@
+    async fn cleanup_backup(&self, backup_info: &str) {}
+
+    /// 迁移失败时保留备份
+    async fn preserve_backup(&self, backup_info: &str, error: &str) {}
+}
+```
+
+提供 `NoopStartupHooks` 用于不需要备份的前端（如数据库后端）。
+
+## 启动流程
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)